src/oom/oomd-manager.c

```diff
@@ -59,6 +61,7 @@ static int process_managed_oom_message(Manager *m, uid_t uid, sd_json_variant *p
                 OomdCGroupContext *ctx;
```

Comment:
pretty sure message should be initialized to
```c
_cleanup_(managed_oom_message_destroy) ManagedOOMMessage message = {
        .duration = USEC_INFINITY,
};
```

simply to stick to the pattern that duration set to USEC_INFINITY means "unset"